.github/workflows/master_push.yml

```diff
@@ -17,18 +31,47 @@ jobs:
   tag-build-push:
     runs-on: ubuntu-latest
     steps:
+      - name: Set part of semantic version to bump
+        id: controls
+        run: |
+          SEMVER_PART=""
+          CHECKOUT_BRANCH="$GITHUB_REF"
+          if ${{github.event_name == 'push' }}; then
+            SEMVER_PART="patch"
+          elif ${{github.event_name == 'workflow_dispatch' }}; then
+            SEMVER_PART=${{ github.event.inputs.bump }}
+            CHECKOUT_BRANCH=${{ github.event.inputs.branch }}
+          fi
+          echo ::set-output name=semver-part::$SEMVER_PART
+          echo ::set-output name=checkout-branch::$CHECKOUT_BRANCH
       - name: Checkout current code
         uses: actions/checkout@master
-      - name: Bump version and push tag
+        with:
+          ref: ${{ steps.controls.outputs.checkout-branch }}
+      - name: Skip version bump merges
+        id: skiptest
+        uses: ./.github/actions/bump-skip
+        with:
+          event-name: ${{ github.event_name }}
+      - name: Bump the tag to a new version
+        if: steps.skiptest.outputs.is-bump == 'no'
+        uses: databiosphere/github-actions/actions/bumper@bumper-0.0.6
         id: tag
-        uses: broadinstitute/github-tag-action@master
         env:
+          DEFAULT_BUMP: patch
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          WITH_V: false
+          HOTFIX_BRANCHES: hotfix.*
+          OVERRIDE_BUMP: ${{ steps.controls.outputs.semver-part }}
+          RELEASE_BRANCHES: master
+          VERSION_FILE_PATH: settings.gradle
+          VERSION_LINE_MATCH: "^gradle.ext.janitorVersion\\s*=\\s*\".*\""
+          VERSION_SUFFIX: SNAPSHOT
       - name: Pull Vault image
+        if: steps.skiptest.outputs.is-bump == 'no'
         run: docker pull vault:1.1.0
       # Currently, there's no way to add capabilities to Docker actions on Git, and Vault needs IPC_LOCK to run.
       - name: Get Vault token
```

Comment:
It would be more in line with current best practices to use github secrets.

Reply:
Agreed, though I think it's worth doing that migration separately